src/components/DonationCard/DonationCard.module.css

```diff
@@ -1,6 +1,96 @@
 .donationCard {
-   border: 1px solid #ccc;
+   margin-bottom: 1.5em;
+   padding: 2em;
+   background-color: var(--bg-primary);
+   border: 1px solid var(--border-color);
    border-radius: 0.7em;
-   padding: 0.7em;
-   margin-bottom: 0.7em;
+   box-shadow: 0 2px 5px var(--shadow-color);
+   cursor: pointer;
+   transition: all 0.2s ease-in-out;
+}
+
+.donationCard:hover {
+   transform: translateY(-5px);
+   box-shadow: 0 4px 10px var(--shadow-color-hover);
+}
+
+.header {
+   display: flex;
+   justify-content: space-between;
+   align-items: center;
+   user-select: none;
+}
+
+.title {
+   color: var(--title-color);
+}
+
+.button {
+   padding: 0.625em 1.25em;
+   border: none;
+   border-radius: 0.7em;
+   background-color: var(--button-bg-color);
+   color: white;
+   cursor: pointer;
+   transition: background-color 0.3s ease;
+}
+
+.button:hover {
+   background-color: var(--button-bg-color-hover);
+}
+
+.description {
+   width: 60%;
```

Comment:
Is the width on purpose? (really asking if this is exactly what you want)

Reply:
yes :)